bs-react-native-jsx3-compat/src/components/touchableWithoutFeedback.re

```diff
@@ -22,43 +20,34 @@ let make =
       ~onPressIn=?,
       ~onPressOut=?,
       ~pressRetentionOffset=?,
-      ~style=?,
```

Comment:
why remove `style`?

Reply:
It doesn't exist for this component https://github.com/facebook/react-native/blob/4900077e83ec7597b2ace3c3882722e752ae64e6/Libraries/Components/Touchable/TouchableWithoutFeedback.js#L56-L81
TouchableWithoutFeedback is cloning its only child so you can pass style there https://github.com/facebook/react-native/blob/4900077e83ec7597b2ace3c3882722e752ae64e6/Libraries/Components/Touchable/TouchableWithoutFeedback.js#L254

Reply:
aah got it! I wasn't aware of that, thanks!

Reply:
Me neither (before implementing that) haha

Reply:
haha